ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/MicrosoftSentinelForm.tsx

```diff
@@ -0,0 +1,529 @@
+                        isRequired
+                        fieldId="notifier.microsoftSentinel.logIngestionEndpoint"
+                        touched={touched}
+                        helperText="(example, https://example-sentinel-ou812.eastus-1.ingest.monitor.azure.com)"
+                        errors={errors}
+                    >
+                        <TextInput
+                            isRequired
+                            type="text"
+                            id="notifier.microsoftSentinel.logIngestionEndpoint"
+                            value={values.notifier.microsoftSentinel.logIngestionEndpoint}
+                            onChange={(event, value) => onChange(value, event)}
+                            onBlur={handleBlur}
+                            isDisabled={!isEditable}
+                        />
+                    </FormLabelGroup>
+                    <FormLabelGroup
+                        label="Directory tenant ID"
+                        isRequired
+                        fieldId="notifier.microsoftSentinel.directoryTenantId"
+                        touched={touched}
+                        helperText="(example, 1234abce-1234-abcd-1234-abcd1234abcd)"
+                        errors={errors}
+                    >
+                        <TextInput
+                            isRequired
+                            type="text"
+                            id="notifier.microsoftSentinel.directoryTenantId"
+                            value={values.notifier.microsoftSentinel.directoryTenantId}
+                            onChange={(event, value) => onChange(value, event)}
+                            onBlur={handleBlur}
+                            isDisabled={!isEditable}
+                        />
+                    </FormLabelGroup>
+                    <FormLabelGroup
+                        label="Application client ID"
+                        isRequired
+                        fieldId="notifier.microsoftSentinel.applicationClientId"
+                        touched={touched}
+                        helperText="(example, abcd1234-abcd-1234-abcd-1234abce1234)"
+                        errors={errors}
+                    >
+                        <TextInput
+                            isRequired
+                            type="text"
+                            id="notifier.microsoftSentinel.applicationClientId"
+                            value={values.notifier.microsoftSentinel.applicationClientId}
+                            onChange={(event, value) => onChange(value, event)}
+                            onBlur={handleBlur}
+                            isDisabled={!isEditable}
+                        />
+                    </FormLabelGroup>
+                    <Card isFlat>
+                        <CardTitle>Authentication Method</CardTitle>
+                        <CardBody>
+                            <ToggleGroup aria-label="Default with single selectable">
```

Comment:
Would this be a better accessible label?
```suggestion
                            <ToggleGroup aria-label="Authentication method selection">
```